crates/proto/proto/zed.proto

```diff
@@ -2251,10 +2251,14 @@ message ContextOperation {
         InsertMessage insert_message = 1;
         UpdateMessage update_message = 2;
         UpdateSummary update_summary = 3;
-        SlashCommandFinished slash_command_finished = 4;
         BufferOperation buffer_operation = 5;
+        SlashCommandStarted slash_command_started = 6;
+        SlashCommandOutputSectionAdded slash_command_output_section_added = 7;
+        SlashCommandCompleted slash_command_completed = 8;
     }
 
+    reserved 4;
```

Comment:
The thought here is that we aren't bothering to make this change fully backward-compatible; if new versions of Zed collaborate with earlier versions of Zed in the assistant panel, the slash commands will just not fully replicate.

We are making sure to prevent decoding errors or type confusion by using new variant numbers for the new slash command messages.